gustaf/spline/base.py

```diff
@@ -681,24 +532,6 @@ def _mfem_ids(self):
 
         return gustaf2mfem, mfem2gustaf
 
-    def extract_bezier_patches(self):
```

Comment:
these functions not needed anymore or did I miss them somewhere?

Reply:
was there to enforce correct return type. updates in splinepy took care of that